ajax/libs/grommet/package.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "grommet",
+  "filename": "Grommet.min.js",
+  "npmName": "grommet",
+  "npmFileMap": [
+    {
+      "basePath": "components",
+      "files": [
+        "**/*"
+      ]
```

Comment:
There are so many files under its `components` folder, if you use `"**/*"` to include all the files, why did you only 3 files in your commit?